### PR TITLE
Hackathon Section added in Expereince Page

### PR DESCRIPTION
**Pull request by RituCs**
Fixed issue #748 and fulfilled all the requirements.

### CODE REVIEW

**Comment by smaranjitghose** [comment]
- Highly unresponsive in mobile screens
- For laptop screens, it almost sticks over the footer
- Very little space between the SubHeading and the Carousel
>The Color of the scrollbar of the timeline should be the same as that of the headings of the page
- I had specifically mentioned the above point so that this addition does not look weird

**Comment by RituCs** [comment]
>     * Highly unresponsive in mobile screens
Done
> 
> 
> mobile_view
> 
>     * For laptop screens, it almost sticks over the footer
Done
> 
>     * Very little space between the SubHeading and the Carousel
is it Heading or SubHeading ? I don't get this . Please explain this.
> 
> 
> > The Color of the scrollbar of the timeline should be the same as that of the headings of the page
> 
>     * I had specifically mentioned the above point so that this addition does not look weird
Sorry for the color of scrollbar. is this #7E7E7E or #0E49B5 ? I am confused what's the exact heading of the page.
Thanks!

**Comment by RituCs** [comment]
Screenshot from 2021-07-01 23-35-43
Please check this . Is it ok ? I already asked my doubt but tried to fix as per my understanding. Thanks!

**Comment by smaranjitghose** [comment]
> > ```
> > * Highly unresponsive in mobile screens
> > ```
> 
> Done
> 
> > mobile_view
> > ```
> > * For laptop screens, it almost sticks over the footer
> > ```
> 
> Done
> 
> > ```
> > * Very little space between the SubHeading and the Carousel
> > ```
> 
> is it Heading or SubHeading ? I don't get this . Please explain this.
> 
> > > The Color of the scrollbar of the timeline should be the same as that of the headings of the page
> > 
> > 
> > ```
> > * I had specifically mentioned the above point so that this addition does not look weird
> > ```
> 
> Sorry for the color of scrollbar. is this #7E7E7E or #0E49B5 ? I am confused what's the exact heading of the page.
> Thanks!



- The subheading "Hackathon Experience",  Heading is always the one on the top like "My Experience"
- Inspect and check the current version whenever in doubt😅. #4e00bb

**Comment by smaranjitghose** [comment]
> Screenshot from 2021-07-01 23-35-43
> Please check this . Is it ok ? I already asked my doubt but tried to fix as per my understanding. Thanks!

Ya. Push the changes in a squash commit. I'll review it

**Review by smaranjitghose — APPROVED** [review]

**Comment by smaranjitghose** [comment]
-  The subheading gets hidden behind the carousel in mobile screens:
Experience_Page

------------------------------------------
------------------------------------------

dd

- Change the color of the buttons and toggle/scroll to that of the scroll bar [here](https://portfolio.smaranjitghose.com/)


- Remove the orange layer over the images

Experience_Page_2

- Put some comments in the CSS file to improve the documentation of your code.

Please do incorporate these basic changes asap and tag me. The rest of the PR is awesome. Looking forward to merging this soon enough!

**Comment by RituCs** [comment]
Screenshot from 2021-07-03 15-39-01
>     * Change the color of the buttons and toggle/scroll to that of the scroll bar [here](https://portfolio.smaranjitghose.com/)
I have changed the color of the buttons but again I could not understand which  toggle/scroll you want to mention .Is it  header section's toggle?
@smaranjitghose

**Comment by smaranjitghose** [comment]
This works. Make the changes

**Comment by RituCs** [comment]
> This works. Make the changes

Done @smaranjitghose .

File: assets/css/experience.css [hunk 1]
```diff
@@ -1,3 +1,4 @@
+@import url('https://fonts.googleapis.com/css?family=Fira+Sans:400,500,600,700,800');
```
Comment (smaranjitghose) [inline assets/css/experience.css:1]: Use Poppins only

**Pull request merged**